.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
please add to your repo file .env-example with placeholders for your sensitive data
my example


.env-example
GOOGLE_CLIENT_ID='<YOUR_GOOGLE_CLIENT_ID>.apps.googleusercontent.com'
GOOGLE_CLIENT_SECRET='<YOUR_GOOGLE_CLIENT_SECRET_KEY>'
EXPRESS_SESSION_KEY='<SESSION_SECRET_KEY>'
USER_ID='<YOUR_USER_ID/>'

# Get SENDGRID API KEY AT https://app.sendgrid.com/settings/api_keys
SENDGRID_API_KEY=<SENDGRID_API_KEY>

Reply:
Done